From 6ca8b28db1c4cb9dc1f18b1ac44d243e3273129d Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 8 May 2018 09:49:08 +0800
Subject: [PATCH 3719/5242] MLK-18218 ARM: dts: imx7ulp-evk: delete property
 to support SD3.0

commit  253206b9ab579e53ca8c415f89d4439ec57a4a1f from
https://source.codeaurora.org/external/imx/linux-imx.git

commit b62dd733a100 ("MLK-18127 ARM: dts: imx7ulp-evk: few correction
for usdhc1") add property "no-1-8-v" for the usdhc1 which limit the
wifi. The sd slot on base board share this usdhc1, so the usdhc1
in imx7ulp-evk-sd1.dts also inherit this property.

delete the "no-1-8-v" property, then the sd slot can support SD3.0

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-sd1.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
index c58e8e0..154b786 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
@@ -42,5 +42,6 @@
 	/delete-property/pm-ignore-notify;
 	/delete-property/keep-power-in-suspend;
 	/delete-property/non-removable;
+	/delete-property/no-1-8-v;
 	status = "okay";
 };
-- 
1.7.9.5

